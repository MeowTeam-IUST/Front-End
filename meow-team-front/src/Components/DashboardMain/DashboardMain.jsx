import React from 'react'
import styles from './DashboardMain.module.scss'
import {EditProfile} from '../EditProfile/EditProfile'
export default function DashboardMain({children}) {
  return (
    <div className={styles.MainContent}>
        {children}
    </div>
  )
}
