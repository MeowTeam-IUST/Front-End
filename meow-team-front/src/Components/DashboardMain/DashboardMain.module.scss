@import "../../App.scss";
.MainContent{
    display: flex;
    height: fit-content;
    padding: 25px 32px 64px 32px;
    flex-direction: column;
    align-items: center;
    gap: 25px;
    flex: 1 0 0;
    border-radius: 38px;
    background: #FFF;

    /* User Shadow */
    box-shadow: 10px 10px 40px 0px rgba(0, 0, 0, 0.15);
}
