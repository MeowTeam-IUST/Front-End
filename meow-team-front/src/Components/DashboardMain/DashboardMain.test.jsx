import { test } from "vitest";
import { render } from "@testing-library/react";
import DashboardMain from "./DashboardMain";

test("DashboardMain component renders children correctly", () => {
  const { asFragment } = render(
    <DashboardMain>
      <div className="child">Child Component</div>
    </DashboardMain>
  );

  expect(asFragment()).toMatchSnapshot();
});
