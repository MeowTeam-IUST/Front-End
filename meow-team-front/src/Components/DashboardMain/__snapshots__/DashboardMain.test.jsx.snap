// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`DashboardMain component renders children correctly 1`] = `
<DocumentFragment>
  <div
    class="_MainContent_01f425"
  >
    <div
      class="child"
    >
      Child Component
    </div>
  </div>
</DocumentFragment>
`;
